src/Authenticators/LDAPAuthenticator.php

```diff
@@ -163,18 +166,39 @@ public static function authenticate($data, Form $form = null)
      * Try to authenticate using the fallback authenticator.
      *
      * @param array $data
-     * @param null|Form $form
+     * @param HTTPRequest $request
      * @return null|Member
+     * @internal param null|Form $form
+     */
+    protected function fallbackAuthenticate($data, HTTPRequest $request)
+    {
+        $authenticatorClass = Config::inst()->get(self::class, 'fallback_authenticator_class');
+        if ($authenticator = Injector::inst()->get($authenticatorClass)) {
+            $result = call_user_func(
+                [
+                    $authenticator,
+                    'authenticate'
+                ],
+                $data,
+                $request
+            );
+            return $result;
+        }
+    }
+
+    /**
+     * @inheritdoc
+     */
```

Comment:
You can just omit this if there's nothing to add, it will be inherited by default